README.md

```diff
@@ -70,6 +70,17 @@ Next we'll workshop Tweets and social media images. Feel free to leave this to u
 
 
 ## Appendix
+### Code Review
+Most Dev.to posts involve code in the form of:
+- Embedded Glitch apps
+- Code snippets in Markdown 
+
+We do not have a strict code review process or standards, but we like to see:
+- well-commented code
+- a README.md in all Glitch apps
```

Comment:
I'd recommend a LICENSE file (probably MIT) be added, too.

Reply:
@AngeloStavrow This is a bigger question for the group...but I wonder if we should go back through all the starters we made for `/@guides` and add licenses?

Reply:
IMHO yes! I think there are a few things to check for on those starters, like making sure they've been added to the `@guides` team and that they have a license file.